FirebaseCore.podspec

```diff
@@ -47,8 +47,8 @@ Firebase Core includes FIRApp and FIROptions which provide central configuration
   s.watchos.framework = 'WatchKit'
 
   # Remember to also update version in `cmake/external/GoogleUtilities.cmake`
```

Comment:
On second thought, I think it would be safer to do this update since Core is going from 7.8 to 7.12

Reply:
Agreed. Running CI in https://github.com/firebase/firebase-ios-sdk/pull/12069 now.